CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # web-ledger-client ChangeLog
 
+## 3.3.1 -
+
+### Added
```

Comment:
For a patch release, and this particular patch, this would be Fixed I think.  Added would typically be associated with a new API or some new functionality, not a bug fix, which would be a minor release.
```suggestion
### Fixed
```